packages/vite/src/node/plugins/css.ts

```diff
@@ -175,6 +175,11 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
         moduleCache.set(id, modules)
       }
 
+      // track deps for build watch mode
+      if (config.command === 'build' && config.build.watch && deps) {
+        deps.forEach(this.addWatchFile)
```

Comment:
I think we should use for-of here to respect the code style in line 205 (that is doing the same).
```suggestion
        for (const file of deps) {
          this.addWatchFile(file)
        }
```

Reply:
OK done.